app/views/groups/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<header class="navbar">
+	<%= button_to "Sign Out", destroy_user_session_path, method: :delete, class: "sign-out-btn"%>
+	<h2 >Categories</h2>	
+</header>
+
+<div class="categories">
+	<ul  >
+		<% @groups.each do |category| %>
+		 <li class="category" >
+			  <%= image_tag category.icon, height: 50 , alt: 'icon'%> 
+			 <div class="category-info">
+				 <%= link_to category.name.capitalize, user_group_path(current_user, category)%>
+				 <p><span> $</span><%= category.purchases.sum { |item| item.entity.amount } %></p>
+			 </div>
+			 <%= button_to 'Delete', user_group_path(current_user, category), method: :delete, class: "delete-btn" %>
+		 </li>
+		<% end %>
+	</ul>
+	<div class="new-category" >
+		<%= link_to 'Add a new category', new_user_group_path(current_user) %>
```

Comment:
- [ ] Kindly use the same color of the design for the button, as you have the colors in the design.

![image](https://user-images.githubusercontent.com/62290545/235161085-3ca1af0e-649d-44b7-b9da-d04d17a95267.png)